src/ui/SimpleFilter/SimpleFilter.ts

```diff
@@ -458,6 +470,21 @@ export class SimpleFilter extends Component {
     return this.circleElement.el;
   }
 
+  public buildClearElement(): HTMLElement {
+    this.clearElement = $$('button', { className: 'coveo-simplefilter-eraser' }, SVGIcons.icons.mainClear);
+    this.clearElement.hide();
+
+    this.clearElement.setAttribute('aria-label', l('Clear', this.options.title));
+    this.clearElement.setAttribute('title', l('Clear', this.options.title));
```

Comment:
Setting the aria-label and title to the same value results in a screen-reader message that is not very nice. It reads "Clear <title> button clear <title>". The `title` should contain additional info for the user. e.g. "deselect all active values in the filter".

Reply:
I took that from `DynamicFacetHeaderButton` so maybe it's needing this adjustment too.

Reply:
Weird, Voice Over doesn't do that

Reply:
I've just tried Voice Over and if the aria-label is the same as title, it reads the label once. But if they're different then it reads the label and then the title. So it does make sense what @samisayegh pointed out.

Reply:
Good to know!